### PR TITLE
Add tree route for display

### PR DESCRIPTION
**Pull request by mboudet**

### CODE REVIEW

File: baricadr/model/backends.py [hunk 1]
```diff
@@ -184,6 +190,68 @@ def restricted_walk(self, path, max_depth):
...
+
+                if file_path.endswith('.rclonelink'):
+                    file_path = file_path[:-11]
+
+                full_file_path = os.path.join(path, file_path)
+
+                if os.path.exists(full_file_path):
+                    remote_list.append({'Path': file_path})
+                else:
+                    remote_list.append({'Path': file_path + "*"})
```
Comment (abretaud) [inline baricadr/model/backends.py:245]: I'd add a space between filename and *

**Comment by abretaud** [comment]
Can you update the changelog while you're at it? (here and barique too I guess)

File: baricadr/model/backends.py [hunk 2]
```diff
@@ -184,6 +190,68 @@ def restricted_walk(self, path, max_depth):
...
+
+                if file_path.endswith('.rclonelink'):
+                    file_path = file_path[:-11]
+
+                full_file_path = os.path.join(path, file_path)
+
+                if os.path.exists(full_file_path):
+                    remote_list.append({'Path': file_path, 'missing': False})
+                else:
+                    remote_list.append({'Path': file_path, 'missing': True})
```
Comment (abretaud) [inline baricadr/model/backends.py:245]: Look at zat, 3 lines economised, make the planet great again

```suggestion
                remote_list.append({'Path': file_path, 'missing': not os.path.exists(full_file_path)})
```

**Comment by mboudet** [comment]
Incredibeul

**Pull request merged**